compiler/luci/pass/src/QuantizeWeightsWithGPTQPass.cpp

```diff
@@ -250,8 +251,18 @@ void cal_minmax_per_channel(CircleConst *node, std::vector<float> &min, std::vec
   iterate_per_channel_with_order(node, cal_minmax, false);
 }
 
-void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t &zp,
-                           float &nudged_min, float &nudged_max, int32_t k_max_scale)
+/**
+ * @brief Compute the scale and zero point for the given range of values
+ * @param min: The minimum value in the range of values to be quantized.
+ * @param max: The maximum value in the range of values to be quantized.
+ * @param k_max_scale: The maximum value of the quantization scale.
+ * @param scaling_factor: The computed scaling factor for the quantization.
+ * @param zp: The computed zero point for the quantization.
+ * @param nudged_min: The nudged minimum value after applying the scaling factor and zero point.
+ * @param nudged_max: The nudged maximum value after applying the scaling factor and zero point.
+ */
```

Comment:
I mean, it's ok to add explanations for each arguments but as other functions don't provide this, so we don't need to add helps like this.
If you really want to add this much of comments, it would be better add to others too... but I don't recommend that.

As I wrote, some arguments are input and some are outputs and this can't be identified at sight when you previously had `k_max_scale` at the end but it was used as input.

Please read my comments carefully.

I recommend to remove all these comments if not really necessary understanding this function.